### PR TITLE
Add a method of checking if running in a container

### PR DESCRIPTION
**Pull request by artificial-aidan**
Fix formatting and typo in readme

Cleaning up isRunningInContainer rule, also add the ability to run a shell command

@GabiGrin this is a pretty big change, feel free to reject or modify. I mostly did this for my own usage, but would be nice to upstream to share easier than passing around a .VSIX file

### CODE REVIEW

**Comment by GabiGrin** [comment]
Hey @artificial-aidan, great addition!
Running a shell command is indeed a powerful action
Curious about your usage of is running in docker :) I also wonder if that could also be generalized to "shell command condition" and run a shell command that expects a boolean return value, as this condition as it feels a bit niche
Merging away as is for the value it brings 

Thanks ⭐ ⭐

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "auto-run-command",
     "displayName": "Auto Run Command",
     "description": "Run a vscode command on project init. Made to run wallaby.js automatically when a project is opened",
-    "version": "1.5.0",
+    "version": "1.5.1",
```
Comment (GabiGrin) [inline package.json:5]:
```suggestion
    "version": "1.6.0",
```

**Comment by GabiGrin** [comment]
Hey @artificial-aidan, great addition!
Running a shell command is indeed a powerful action
Curious about your usage of is running in docker :) I also wonder if that could also be generalized to "shell command condition" and run a shell command that expects a boolean return value, this condition as it feels a bit niche
Merging away as is for the value it brings 

Thanks ⭐ ⭐

**Comment by artificial-aidan** [comment]
Thanks for merging! How long until its updated on the vscode marketplace?

Also, looks like this closes a few issues.

**Comment by GabiGrin** [comment]
@artificial-aidan  1.6.0 is available now :)

**Pull request merged**